cclib/parser/gaussianparser.py

```diff
@@ -2045,3 +2066,23 @@ def natural_orbital_single_spin_parsing(inputfile, updateprogress_title):
 
         if line[:31] == ' Normal termination of Gaussian':
             self.metadata['success'] = True
+
+        # define some extra properties that can be approximated by homo and lumo energies
+        # definitions: https://pubs.acs.org/doi/10.1021/tx2003257
+        # derivations: https://www.mdpi.com/1422-0067/5/8/239/html)
+        if hasattr(self, "homoenergies") and hasattr(self, "lumoenergies"):
+            self.hardness = []
+            self.softness = []
+            self.chempot = []
+            self.electrophilicity = []
+            for i in range(0,len(self.homoenergies)):
+                homoenergies = self.homoenergies[i]
+                lumoenergies = self.lumoenergies[i]
+                hardness = (lumoenergies - homoenergies) / 2
+                softness = 1 / hardness
+                chempot = (lumoenergies + homoenergies) / 2
+                electrophilicity = (chempot**2) / (2*hardness)
+                self.hardness.append(hardness)
+                self.softness.append(softness)
+                self.chempot.append(chempot)
+                self.electrophilicity.append(electrophilicity)
```

Comment:
again depending on how `self.homoenergies` looks like, these could all be numpy operations.

Reply:
looking at an example I think this could just be:
```
self.hardness = (self.homoenergies - self.lumoenergies) / 2
etc..
```
as long as you turn self.homoenergies into an array

Reply:
if there is any doubt about homoenergies and lumoenergies being included, then these properties would definitely not make it. They are pretty much approximations that, as with the homo and lumo energy, the user can easily calculate themselves. Approximations doesn't really fit in with the rest of the data cclib extracts, i.e. exact values taken directly from the output files.